src/main/java/net/sf/jabref/logic/exporter/ExportFormat.java

```diff
@@ -78,6 +79,24 @@ public ExportFormat(String displayName, String consoleName, String lfFileName, S
     }
 
     /**
+     * Initialize another export format based on templates stored in dir with
+     * layoutFile lfFilename.
+     *
+     * @param displayName Name to display to the user.
+     * @param consoleName Name to call this format in the console.
+     * @param lfFileName  Name of the main layout file.
+     * @param directory   Directory in which to find the layout file.
+     * @param extension   Should contain the . (for instance .txt).
+     * @param layoutPreferences Preferences for layout
+     */
```

Comment:
savePreferences should be described and stated for what they are used in the formatter.